src/less/icon/common/ds6/icon-mixin.less

```diff
@@ -0,0 +1,3 @@
+.icon-mixin() {
+    @import "../../../../../dist/icon/ds6/icon.css";
```

Comment:
Gah! @seangates  Looks like we both missed this. Will have to put out a patch to fix the now broken `gh.browser.json`.